DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/README.md

```diff
@@ -0,0 +1,317 @@
+   
+   ```verilog
+   add_fpga_ip_export_1_di add_fpga_ip_export_1_di_inst (
+      // Interface: clock (clock end)
+      .clock                          ( ), // 1-bit clk input
+      // Interface: resetn (reset end)
+      .resetn                         ( ), // 1-bit reset_n input
+      // Interface: device_exception_bus (conduit end)
+      .device_exception_bus           ( ), // 64-bit data output
+      // Interface: kernel_irqs (interrupt end)
+      .kernel_irqs                    ( ), // 1-bit irq output
+      // Interface: csr_ring_root_avs (avalon end)
+      .csr_ring_root_avs_read         ( ), // 1-bit read input
+      .csr_ring_root_avs_readdata     ( ), // 64-bit readdata output
+      .csr_ring_root_avs_readdatavalid( ), // 1-bit readdatavalid output
+      .csr_ring_root_avs_write        ( ), // 1-bit write input
+      .csr_ring_root_avs_writedata    ( ), // 64-bit writedata input
+      .csr_ring_root_avs_address      ( ), // 5-bit address input
+      .csr_ring_root_avs_byteenable   ( ), // 8-bit byteenable input
+      .csr_ring_root_avs_waitrequest  ( )  // 1-bit waitrequest output
+   );
+   ```
+
+   Linux:
+
+   ```
+   $> cd .. # navigate to project root if not there already
+   $> cp -r add-oneapi/build/add.fpga_ip_export.prj_1/ add-quartus/
+   ```
+
+   Windows:
+
+   ```
+   > cd .. # navigate to project root if not there already
+   > xcopy add-oneapi\build\add.fpga_ip_export.prj_1\ add-quartus\add.fpga_ip_export.prj_1 /e /s /i
+   ```
+
+4. Create the Platform Designer system.
+
+   1. Open Platform Designer from the Intel® Quartus® Prime GUI:
+
+      ![](assets/open-platform-designer-button.png)
+
+      Create a new system by clicking the 'New Platform Designer System' button (![](assets/new-platform-designer-system-button.png)) and name it `add_kernel_wrapper.qsys`.
+
+      Configure the `Reset Bridge` IP as shown:
+
+      ![](assets/reset-bridge.png)
+
+   2. Add the following IP to your system:
+
+      * Basic Functions > Bridges and Adaptors > Memory Mapped > **JTAG to Avalon Master Bridge Intel® FPGA IP**
+
+      * oneAPI > **add_fpga_ip_export_1_di**
+
+      ![](assets/add-ip-platform-designer.png)
+
+   3. Connect the modules as shown:
+
+      ![](assets/complete-system_platform-designer.png)
+
+      Don't forget to export the `irq_add` and `exception_add` signals. We provided a top-level RTL file (`add.sv`) that uses the generated IP. Following these naming conventions allows you to connect the oneAPI kernel to this handwritten RTL.
+
+      > **Note**: There is a bug in oneAPI 2023.1 that causes the generated IP to be incorrectly documented in its hardware TCL script. You can override this in platform designer by changing the signal type of the `resetn` signal to `reset_n`:
```

Comment:
Will you update this tutorial when the bug is fixed? Do we know when it will be fixed? It might be better to call out the version:

**Important**: If you are using the oneAPI Base Toolkit 2023.1, the DCP++/C++ compiler causes the generated IP to be incorrectly documented in its hardware TCL script. You can override this in platform designer by changing the signal type of the `resetn` signal to `reset_n`: